valhalla/sif/dynamiccost.h

```diff
@@ -997,6 +1000,8 @@ class DynamicCost {
   bool shortest_;
 
   bool ignore_restrictions_{false};
+  bool ignore_non_vehicular_restrictions_{false};
+  bool ignore_turn_restrictions_{false};
```

Comment:
we should add a quick note that it's not a request option, just a combination of the 2 above.